lib/context.js

```diff
@@ -426,6 +426,17 @@ api.createTermDefinition = (activeCtx, localCtx, term, defined) => {
     mapping['@container'] = container;
   }
 
+  // scoped contexts
+  if('@context' in value) {
+    if(activeCtx.processingMode && activeCtx.processingMode.indexOf('json-ld-1.1') !== 0) {
```

Comment:
If we're going to have a decent number of these checks, we should probably make a helper function that takes something like the active context, the processing mode required, and optional error text+details to use if the check fails.

Reply:
The processingMode check can be replaced with the separate check-td-keys, which will simplify these tests, but a helper function is probably worthwhile.